webapp/channels/src/components/widgets/inputs/input/input.test.tsx

```diff
@@ -31,18 +43,232 @@ describe('components/widgets/inputs/Input', () => {
             />,
         );
 
-        const clear = wrapper.find('.Input__clear');
-        expect(clear.length).toEqual(1);
-        expect(wrapper.find('CloseCircleIcon').length).toEqual(1);
+        // Find the input with the value
+        const inputElement = screen.getByDisplayValue(value);
+        expect(inputElement).toBeInTheDocument();
 
-        const tooltip = wrapper.find(WithTooltip);
-        expect(tooltip.length).toEqual(1);
+        // Find the clear button's div container
+        const iconElement = screen.getByTestId('close-circle-icon');
+        expect(iconElement).toBeInTheDocument();
 
-        const titleProp = tooltip.prop('title');
-        expect(titleProp).toEqual(clearableTooltipText);
-
-        clear.first().simulate('mousedown');
+        // Click directly on the icon element
+        await act(async () => {
+            userEvent.click(iconElement);
+        });
 
+        // Verify onClear was called
         expect(onClear).toHaveBeenCalledTimes(1);
     });
+
+    describe('minLength validation', () => {
```

Comment:
input had only maxLenght, added minlength validation as per required by design

Reply:
Are we actually using the minimum length in this PR or is it just something we might use? I don't see it referenced elsewhere